src/components/Carousel/index.js

```diff
@@ -75,26 +75,63 @@ export default class Carousel extends PureComponent {
     highlightOnHover: false,
     loop: false,
     overflow: false,
-    scrollCount: 1,
+    scrollCount: 2,
     showCount: 3,
     transition: TRANSITION_SLIDE,
   }
 
+  state = {
+    currentSlide: 0,
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.slider = props.sliderRef || React.createRef()
+  }
+
+  handlePrevClick = () => {
+    const { scrollCount } = this.props
+    const { currentSlide } = this.state
+
+    const goTo = currentSlide - scrollCount < 0
+      ? 0
+      : currentSlide - scrollCount
+
+    this.slider.current.slickGoTo(goTo)
+  }
+
+  handleNextClick = () => {
+    const { children, scrollCount, showCount } = this.props
+    const { currentSlide } = this.state
+
+    const goTo =
+      currentSlide + showCount + scrollCount > children.length - 1
+        ? children.length - showCount
+        : currentSlide + scrollCount
+
+    this.slider.current.slickGoTo(goTo)
+  }
+
+  handleAfterChange = (index) => {
+    this.setState({ currentSlide: index })
+  }
+
   render () {
     const {
       autoPlay,
       centered,
       children,
       className,
       controls,
+      focusOnSelect,
       highlightOnActive,
       highlightOnHover,
       loop,
       overflow,
       peek,
       scrollCount,
       showCount,
-      sliderRef,
```

Comment:
Do we need to keep `sliderRef` here to handle multiple carousels controlling one another?

Reply:
Oh looks like it was changed from `sliderRef` to just `this.slider`, maybe this isn't needed then

Reply:
correct.  the ref is stored on the instance in the constructor because there's some logic associated to generating it now.  basically, the arrows need to be able to access the ref, but if one is not passed as a prop, there isn't one.  so we say use the one given or create one.